docs/plugins/fluentbit/clusterinput/node-exporter-metrics.md

```diff
@@ -0,0 +1,40 @@
+# Node Exporter Metrics
+
+A plugin based on Prometheus Node Exporter to collect system / host level metrics.
+
+>Note: Metrics collected with Node Exporter Metrics flow through a separate pipeline from logs and current filters do not operate on top of metrics.
+
+This plugin is currently only supported on Linux based operating systems
+
+
+| Field          | Description                                                                           | Scheme |
+|----------------|---------------------------------------------------------------------------------------|--------|
+| tag            | Tag name associated to all records comming from this plugin.                          | string |
+| scrapeInterval | The rate at which metrics are collected from the host operating system.               | *int32 |
+| path.procfs    | The mount point used to collect process information and metrics, default is `/proc/`. | string |
+| path.sysfs     | The path in the filesystem used to collect system metrics, default is `/sys/`.        | string |
+
+
+Important Note: The input plugin of node exporter metrics will collect system / host level metrics from specified path,
+so we should mount those hostpath to containers. For example, This plugin will mount `/proc/` to collect process information and metrics 
+and `/sys/` to collect system metrics by default if we don't specify path, thus we can add it to `values.yaml` in charts, like:
+
+```yaml
+fluentbit:
```

Comment:
I think we can add a nodeExporterMetrics section here for two purposes:
- Add an example for users who want to collect node exporter metrics
- Redefine the `path.procfs` to `/host/proc`, `path.sysfs` to `/host/sys` , because the default is `/proc` and `/sys`

![image](https://user-images.githubusercontent.com/18525465/178888748-e1d8af74-41b6-4e72-b36c-5f1fedae2d3f.png)

Reply:
> I think we can add a nodeExporterMetrics section here for two purposes:
> 
> * Add an example for users who want to collect node exporter metrics
> * Define the `path.procfs` as `/host/proc`, define `path.sysfs` as `/host/sys` , because the default is `/proc` and `/sys`
> 
> ![image](https://user-images.githubusercontent.com/18525465/178888748-e1d8af74-41b6-4e72-b36c-5f1fedae2d3f.png)

Merge this for now. @wenchajun, would you add this doc for @mangoGoForward ?

Reply:
Sorry for late to reply. I haven't figured out how to improve this doc, and appreciate if someone would like to contribute.

Reply:
I created a PR for this https://github.com/fluent/fluent-operator/pull/353

Reply:
> I created a PR for this #353

Thanks~